docs/docs/30-how-to-guides/30-docker-image.md

```diff
@@ -17,7 +17,9 @@ easiest option for experimenting locally with Kargo Render!
 Example usage:
 
 ```shell
-docker run -it ghcr.io/akuity/kargo-render:v0.1.0-rc.36 \
+docker run -it ghcr.io/akuity/kargo-render:v0.1.0-rc.35 \
+  kargo-render \
+  render \
```

Comment:
Similarly, this won't be required once `.36` is cut with [the change to `entrypoint` over `cmd`](https://github.com/akuity/kargo-render/pull/261/files#diff-dd2c0eb6ea5cfc6c4bd4eac30934e2d5746747af48fef6da689e85b752f39557)